pyHS100/smartplug.py

```diff
@@ -35,7 +35,7 @@ def __init__(self,
                  host: str,
                  protocol: 'TPLinkSmartHomeProtocol' = None) -> None:
         SmartDevice.__init__(self, host, protocol)
-        self.emeter_type = "emeter"
+        self._type = "emeter"
```

Comment:
I don't quite get why this is necessary. Perhaps there are other changes, related, that are not part of this pull yet?

Reply:
I'm not sure either, I think it was added by @newAM

Reply:
git blame shows you added it

Reply:
Maybe you deleted “emeter” by mistake when editing. Let’s revert it if it’s not clear why it was changed.

Reply:
thanks guys, no idea how I did that, I'll put it back.